PatientRegistrationApp/src/components/HouseholdContactInfo.vue

```diff
@@ -1,113 +1,99 @@
 <template>
 	<v-container fluid>
-		<v-row>
-		<v-radio-group
-        required
-        :rules="[v => !!v || 'This field is required']"
-        v-model="phoneNumberAvailableRadioButtons"
-		>
-        <v-col align="right" cols="12">
-          <v-radio label="I have a phone number" value="yes" @change="PhoneNumberAvailable()"></v-radio>
-          <v-radio label="I do not have a phone number" value="no" @change="PhoneNumberNotAvailable()"></v-radio>
-		</v-col>
-		</v-radio-group>
+		<v-row align="center" justify="start">
+			<v-col cols="12">
+				<div>We ask that you please provide your phone number and/or e-mail address.  This contact information will provide us with the ability to send you important information regarding your vaccination registration, follow-up appointment, and vaccination summary.</div>  
+				<div><br>By withholding your e-mail and/or phone number, it will not be possible to provide you with the important vaccination information.</div>
+				<div><strong>We strongly encourage you to provide your e-mail and/or phone number as contact information.</strong></div>
+				<div><br></div>
+				<v-divider></v-divider>
+				<div><br>By providing your e-mail and/or phone number as contact information, you agree to the following:</div>
+				<div><strong>You understand that we will use the phone number and e-mail that you provide to contact you with vaccination registration and follow-up information.</strong></div>
```

Comment:
I asked my Mom who teaches middle school to help me get this down to a 6th grade level of reading, this is what we came up with:

We ask that you please provide your phone number and/or e-mail address. This will make it possible for us to send you information about your vaccination registration, follow-up appointment, and vaccination summary.

This information will allow us to follow up on any details that need to be handled quickly.
We strongly encourage you to provide your e-mail and/or phone number as contact information.

________________________________________

By providing your e-mail and/or phone number as contact information, you agree to the following:
You understand that we will use the phone number and e-mail that you provide to contact you with vaccination registration and follow-up information.

Reply:
@TomMakes I'd say we can leave this as is. This is legal jargon for permission to contact that will have to be set by the client at the end of the day so being readable by a 6th grader isn't actually a goal (I mean what 6th grader can read a terms and conditions page and understand it)... 

I would counter if we want to put this into the separate file for CustomerSettings or something, maybe this would be a good candidate for that as well?

Reply:
Teri's guidance was to ensure that we keep the language at a 6th grade reading level, as that will be most accessible for the general population (we don't intend for actual 6th graders to read this, but many adults read at that 6th grade level).

I created a customerSettings.js configuration file and modified the code to retrieve the strings from that file.  I included our original wording as the default, but also included Tom's suggestion as an alternative (commented out, but is there for an alternative suggestion)